### DISCUSSION TITLE
Is there anything you need tested?

### DISCUSSION THREAD

**Discussion by ShaunTheSalvo** [shadichy/systemback-archlinux#12] [General]
Hi Shadichy,

I've noticed you've made a few changes to the source in the last week or so. Just curious if there's anything you'd like tested?

Take care.

Cheers - Shaun.

**Comment by shadichy** [comment]
Hi Shaun,

Thanks for your question!

Actually, systemback-cli is not working as I expected, and some features like scheduler need to be tested
Furthermore, I'm planning to make a port for `dracut` alongside `mkinitcpio` one

Hope you have a nice day

Shadichy.

**Reply by ShaunTheSalvo** [reply]
Well, I'm certainly happy to help with bug-tracking if you like.

Just tried to run systemback-cli (built new copy from source today) and - yep, not very interesting. :) Just a garbage output at the top of the terminal (0jM-;M-RU0pM-;M-RU^PM-DM-RU - if that's of any help) and then it freezes.

Let me know if you have anything you'd like me to test. Happy to be a guinea pig, alongside my friend Gene.

All the best from Australia!

Shaun & Gene.
